packages/Liquid-Core.package/LQPoll.class/instance/returnDataWithDelimiter..st

```diff
@@ -0,0 +1,6 @@
+printing
+returnDataWithDelimiter: aDelimiter 
+	| aStream |
+	aStream := String new writeStream.
+	self printDataOn: aStream withDelimiter:';'.
```

Comment:
Using the symbol notation (`$;`) would be preferred for single chars.

However, we can define our own guidelines and use `';'` instead.